modules/govuk_crawler/templates/govuk_sync_mirror.erb

```diff
@@ -37,9 +39,9 @@ function get_error_page() {
   # Grab the 503 error page
   log "user.info" "Grabbing the latest error page"
 
-  website_mirror_dir="${MIRROR_ROOT}"/www.*
-  mkdir -p "${website_mirror_dir}/error/"
-  curl -sf "https://${STATIC_HOSTNAME}/templates/503.html.erb" -o "${website_mirror_dir}/error/503.html"
+  website_mirror_dir="${MIRROR_ROOT}"/"${GOVUK_WEBSITE_DIR}"
```

Comment:
Nice. You could get rid of a couple of quotes here:

```suggestion
  website_mirror_dir="${MIRROR_ROOT}/${GOVUK_WEBSITE_DIR}"
```

Reply:
Thanks for that, fixed now 👍